src/sales/sales-menu.md

```diff
@@ -22,45 +22,45 @@ On the _Admin_ sidebar, click **Sales**.
 ## Menu Options
 
 <!--{% if "Default.B2B Only" contains site.edition %}-->
-### [Quotes]({{ site.baseurl }}{% link sales/quotes.md %})
+### Quotes
 
 Authorized buyers can negotiation the price with the seller by sending a [request]({{ site.baseurl }}{% link sales/quote-price-negotiation.md %}) from the shopping cart.
 <!--{% endif %}-->
-### [Orders]({{ site.baseurl }}{% link sales/orders.md %})
+### Orders
 
-When an order is placed, a sales order is created as a temporary record of the transaction. Payment has not been processed, and the order can still be canceled.
+When an [order]({{ site.baseurl }}{% link sales/orders.md %}) is placed, a sales order is created as a temporary record of the transaction. Payment has not been processed, and the order can still be canceled.
 
-### [Invoices]({{ site.baseurl }}{% link sales/invoices.md %})
+### Invoices
 
-An invoice is a record of the receipt of payment for an order. Multiple invoices can be created for a single order, each with as many, or as few of the purchased products that you specify. Depending on the payment action, payment can be automatically captured when the invoice is generated.
+An [invoice]({{ site.baseurl }}{% link sales/invoices.md %}) is a record of the receipt of payment for an order. Multiple invoices can be created for a single order, each with as many, or as few of the purchased products that you specify. Depending on the payment action, payment can be automatically captured when the invoice is generated.
 
-### [Shipments]({{ site.baseurl }}{% link sales/shipments.md %})
+### Shipments
 
-A shipment is a record of the products in an order that have been shipped. As with invoices, multiple shipments can be associated with a single order, until all of the products in the order are shipped.
+A [shipment]({{ site.baseurl }}{% link sales/shipments.md %}) is a record of the products in an order that have been shipped. As with invoices, multiple shipments can be associated with a single order, until all of the products in the order are shipped.
 
-### [Dispatches]({{ site.baseurl }}{% link sales/dispatches.md %})
+### Dispatches
```

Comment:
Find a place here to link the `({{ site.baseurl }}{% link sales/dispatches.md %})` page

Reply:
**done**